### DISCUSSION TITLE
MkDocs Material ignores Img height property?

### DISCUSSION THREAD

**Discussion by Andre601** [squidfunk/mkdocs-material#5236] [Q&A]
Right now I encounter an issue where it seems like that Material for MkDocs ignores the `height` property of an `<img>` tag.
Setting the `width` property works perfectly fine tho.

Examples:
![grafik](https://user-images.githubusercontent.com/11576465/226498746-88c7841d-3eb1-42a5-881e-2eb454892ccb.png)
![grafik](https://user-images.githubusercontent.com/11576465/226498821-d5a40668-d3b2-4478-aef9-efe48b709d5d.png)

**Comment by squidfunk** [comment]
Height is set relative to width, as width is normally the limiting factor when viewing web pages:

https://github.com/squidfunk/mkdocs-material/blob/e24759fb41719d00874fc762a3e750ac857eb604/src/assets/stylesheets/main/_typeset.scss#L389-L395

You could add a CSS class that sets`width` to `auto` based on `height`. Also, [`aspect-ratio`](https://caniuse.com/mdn-css_properties_aspect-ratio) might be helpful, but has yet limited browser support. Depending on which browser you want to support, switching to `aspect-ratio` might be a good idea.
